tests/src/test/scala/org/http4s/parser/OriginHeaderSuite.scala

```diff
@@ -0,0 +1,85 @@
+/*
+ * Copyright 2013 http4s.org
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.http4s
+package parser
+
+import cats.data.NonEmptyList
+import org.http4s.headers.Origin
+import org.http4s.syntax.all._
+
+class OriginHeaderSuite extends munit.FunSuite {
```

Comment:
interesting any reasons you stick with `munit.FunSuite` instead of:

```suggestion
class OriginHeaderSuite extends Http4sSuite {
```

Reply:
Well I don't need anything from HttpSuite for this particular test, so it seems less fragile to use the least powerful option that will do the job